consensus/group/genesis.go

```diff
@@ -54,7 +55,7 @@ func GenerateGenesis() *types.GenesisInfo {
 	f := common.GlobalConf.GetSectionManager("consensus").GetString("genesis_group_info", "genesis_group.info")
 	genesis := genGenesisStaticGroupInfo(f)
 	gHeader := &groupHeader{
-		seed:          genesis.Seed,
+		seed:          genGenesisGroupSeed(genesis),
```

Comment:
should specify the seed at the genesis_group.info config file, and the value simply is ByteToHash([]byte("zvchain genesis group"))